modules/client/src/main/java/org/apache/ignite/internal/client/table/ClientKeyValueView.java

```diff
@@ -458,7 +458,7 @@ private void writeKeyValue(ClientSchema s, PayloadOutputChannel w, @Nullable Tra
     }
 
     private void writeKeyValueRaw(ClientSchema s, PayloadOutputChannel w, @NotNull K key, V val) {
-        var builder = BinaryTupleBuilder.create(s.columns().length, true);
+        var builder = new BinaryTupleBuilder(s.columns().length, true);
```

Comment:
Because I needed to inherit from the Builder. I could have made the constructor `protected`, but I really don't see any reason in having a factory method that simply delegates to a constructor